.gitignore

```diff
@@ -25,3 +25,4 @@ node_modules
 npm-debug.log
 keys
 third_party/bower
+bower_components
```

Comment:
The way it is now will ignore any directory and subdirectory named bower_components. You can check that git is ignoring those files in your client.